Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -362,6 +362,12 @@ export type Props = $ReadOnly<{|
    * ```
    */
   contentContainerStyle?: ?ViewStyleProp,
+  /**
+   * When true, the scroll view stops on the index to which the gesture ended.
```

Comment:
@sahrens  I feel like "stops on the next index regardless of how fast the gesture is" could be a bit misleading with this implementation, since one could theoretical scroll from `index` to `index +2` if the scroll distance on gesture release is greater than the width of the item.

What about "stops on the next index (in relation to final scroll position) regardless of how fast the gesture is"?

Reply:
"final" is still a little confusing. Maybe "stops on the next index (in relation to scroll position at release) regardless of how fast the gesture is"?

Reply:
That's solid, I'll update in a bit